### ISSUE TITLE
Employee attendance trackable in hours

### ISSUE THREAD

**Issue by julsmon** [frappe/erpnext#17974]
So far attendance can be tracked only by status (present, half day...). 
In most countries attendance needs to be registered by tracking hours instead, so it would be very useful to add this feature and to allow for an hour-based monthly attendance report.

**Comment by emagray** [comment]
I agree. I have created a custom Doctype as a workaround for now, but this really should be the default behavior in ERPNext.

**Comment by julsmon** [comment]
@emagray Hello and thanks for the back up! I tried a workaround too, by creating a custom doctype, but could not really turn it into a suitable report. I hope the team will consider this feature suggestion over the next releases.
